cloud-formation/scripts/post-dev-stack-creation.sh

```diff
@@ -3,6 +3,8 @@
 source ./utils.sh
 source ./stack-name.sh
 
+export AWS_PROFILE=media-service
```

Comment:
Some people might have multiple profiles they work on - wouldn't this be inconvenient for them?